components/typography/index.css

```diff
@@ -353,7 +353,7 @@ governing permissions and limitations under the License.
 
 /* Windows high contrast mode */
 @media (forced-colors: active) {
-  .spectrum-body {
+  .spectrum-Body {
```

Comment:
Interesting! Great catch

Reply:
We could do the same here, basically keep both selectors and add a `/** @deprecation .spectrum-body will be removed during the S2 migration; please update your code to .spectrum-Body */`

Reply:
I don't think we need to do this. `.spectrum-Body` is correct and is what is used everywhere else in this CSS file. This was the only place with the typo

Reply:
Yeah, I think we're safe on this one without the fallback.